erpnext/stock/doctype/item/item.py

```diff
@@ -958,13 +958,13 @@ def _get_linked_submitted_documents(self, changed_fields: List[str]) -> Optional
 				):
 					return linked_doc
 
-			elif linked_doc := frappe.db.get_value(
-				doctype,
-				filters,
-				["parent as docname", "parenttype as doctype"],
-				as_dict=True,
-			):
-				return linked_doc
+				elif linked_doc := frappe.db.get_value(
+					doctype,
+					filters,
+					["parent as docname", "parenttype as doctype"],
+					as_dict=True,
+				):
+					return linked_doc
```

Comment:
This can be reverted. The `elif` is outside so that it handles:
``` py
if doctype not in ("Purchase Invoice Item","Sales Invoice Item",)
```

From the `linked_doctypes` list only Product Bundle will not have a parent or parenttype.